examples/arbitrary_json.rs

```diff
@@ -0,0 +1,32 @@
+//! This example illustrates the way to send and receive arbitrary JSON.
+//!
+//! This is useful for some ad-hoc experiments and situations when you don't
+//! really care about the structure of the JSON and just need to display it or
+//! process it at runtime.
+extern crate reqwest;
+#[macro_use] extern crate serde_json;
+
+fn main() {
+    let echo_json: serde_json::Value = reqwest::Client::new()
+        .post("https://jsonplaceholder.typicode.com/posts")
+        .body(
```

Comment:
In both of these examples, there's no need for `serde_json`, you can call `post(url).json(serializable_type)` :)

Reply:
Thanks for the hint! I have updated the examples.